.github/workflows/ci.yml

```diff
@@ -8,4 +8,4 @@ on:
 
 jobs:
   build:
-    uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v6
+    uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v5
```

Comment:
automatically reverted by our market/up2datekeeper job.
but it's just right like that, currently test feedback is broken. I can't work without and do manual clicking
https://github.com/axonivy-market/excel-importer/pull/76